### ISSUE TITLE
'spring.cloud.task.initialize.enable' is an unknown property. 

### ISSUE THREAD

**Issue by dlchris** [spring-cloud/spring-cloud-task#584]
hi , I found set the spring.cloud.task.initialize.enable property to false, as follows:
spring.cloud.task.initialize.enable=false

**error : **My sts IDE prompts the 'spring.cloud.task.initialize.enable' is an unknown property.**** 

spring cloud version:
<spring-cloud.version>Greenwich.BUILD-SNAPSHOT</spring-cloud.version>

**Comment by cppwfs** [comment]
Hello dchris,
Can you share the steps you took to get this error?

**Comment by dlchris** [comment]
![image](https://user-images.githubusercontent.com/3303452/55371312-52f3a080-5530-11e9-8bdf-0cc15917d57e.png)

**Comment by dlchris** [comment]
hi cppwfs, ask a question
Use SpringCloudTask, Invoking the remote client service with Feign, Successfully obtained the result , but In the log reports have errors,  as follows:
![image](https://user-images.githubusercontent.com/3303452/55386230-a16d6300-5561-11e9-8b5b-a49b461bc186.png)

source code , as follows:
**Main**
![image](https://user-images.githubusercontent.com/3303452/55386337-df6a8700-5561-11e9-8b9b-2f2999f581dd.png)

**TaskCommondLineRunner**
![image](https://user-images.githubusercontent.com/3303452/55386465-1d67ab00-5562-11e9-8b06-11166e1abd95.png)

**ServiceHiFeign**
![image](https://user-images.githubusercontent.com/3303452/55386540-49832c00-5562-11e9-869d-7c3b1d8f8786.png)

**Comment by cppwfs** [comment]
I identified the issue as to why you are not seeing `initialize.enable `in your properties. I created a issue and reference this one:  https://github.com/spring-cloud/spring-cloud-task/issues/587

To your second question:
I think you maybe using Spring Cloud Task incorrectly.  Spring Cloud Task is meant to be used for short lived processes (Ephemeral apps)  and not LRP (Long Running Processes).  ie. A task is an app that starts, does its work and then terminates like a Spring Batch application or an app that uses  Boot's ApplicationRunner (simple task demoed here: https://github.com/spring-cloud/spring-cloud-task/tree/master/spring-cloud-task-samples/timestamp) .   The app that you are demoing above is a LRP, an app that runs until and external agent (like yourself) terminates the application (ctrl-c).
